### ISSUE TITLE
Bug in tiled map

### ISSUE THREAD

**Issue by EzeBais** [craftworkgames/MonoGame.Extended#418]
Hi everyone. I tried draw a TiledMap in Monogame Extended and I got this error:

![bug](https://user-images.githubusercontent.com/31544294/29976103-4e76b0ee-8f0f-11e7-96e2-758589464607.png)

Any ideas?

The map:
[Mapas.zip](https://github.com/craftworkgames/MonoGame.Extended/files/1270670/Mapas.zip)

**Comment by craftworkgames** [comment]
I had a quick look at the files today.

My first thought was that the texture sizes didn't match with the number of tiles (this kind of thing has proven problematic before) but everything seems to check out okay here.

The next step is to load this map into one of the demos to try and reproduce the problem. I haven't had a chance to do this yet (I'm not on my own computer right now). I'll try find time to do this tonight.

**Comment by craftworkgames** [comment]
Hey,

I loaded your whole map into the demo and I can't see the blue rectangle anywhere. I tried moving around the map and zooming in and out. Nothing out of the ordinary.

![image](https://user-images.githubusercontent.com/3201643/30057765-6d5fd62e-927b-11e7-8655-082c0f310865.png)

I wonder if there's something else at play here? Maybe excess memory usage or a graphics card / driver incompatibility?

I believe you saw a real problem. Unfortunately, it's very difficult to fix something without first reproducing it.
